compiler/pash_compilation_server.py

```diff
@@ -414,6 +429,16 @@ def run(self):
             self.parse_and_run_cmd(input_cmd)
         
         self.connection_manager.close()
+        if not self.parallelized_flag:
+            log("No parts of the input script were parallelized. Ensure commands are annotated for parallelization.")
+        elif all(
+            proc_info.exec_time is not None and proc_info.exec_time < 1
+            for proc_info in self.process_id_input_ir_map.values()
+        ):
+            log("Some script fragments were parallelized, but their execution times were negligible.")
+            log("Consider optimizing your script to include longer-running tasks.")
```

Comment:
No need for this message, this is not really an optimization that we are asking them to do, but rather making sure that they have annotations for the long-running parts that they care about.